tests/clock/jobCleanup.js

```diff
@@ -175,7 +175,7 @@ describe('jobCleanup', () => {
     const batchSize = 5;
 
     it('skip 3', (done) => {
-      const delay = 199;
+      const delay = 190;
```

Comment:
I had set these to 99 because I was seeing flappers where it was returning too few results. Now we're seeing flappers with too many, so I'm putting it down to 90.
In the last changes I made I also significantly increased the duration so the margin of error is much bigger now, so I don't think the original problem will come up.